.dockerignore

```diff
@@ -1 +1,17 @@
-dataset-uploads/
\ No newline at end of file
+# Don't put unnecessary stuff in the image
+
+# Ignore everything by default
+*
+
+# Explicitly add certain files and directories
+!LICENSE
+!README.md
+!assets/
+!bin/
+!demodata.json
```

Comment:
Not sure we need to put this in the image because it'll mainly used by people that have the docker-compose mount.